src/api.py

```diff
@@ -23,97 +21,136 @@ def home():
         return 'WELCOME TO CABINET'
 
 
-    # @app.route('/blob', methods=['GET', 'POST'])
-    # #TODO: learn about bytea postgres type. 
-    # def store_blob():
-    #     if request.method == 'GET':
-    #         raise NotImplementedError
-    #     if request.method == 'POST':
-    #         blob_bytes = request.get_json()['blob_bytes']
-    #         blob_id = f.add_blob(blob_bytes) 
-    #         return json.dumps({'blob_id':blob_id})
-
-    # NOTE: each data type has its own end point with predefined columns
-    # any changes to the table must be done in source code, not by user
-
     @app.route('/blob', methods=['GET', 'POST'])
     def blob(): 
         try:
             conn, cur = db_connect(env=env)
             if request.method == 'GET':
                 user_search = request.args.to_dict()
                 if not 'blob_type' in user_search.keys():
-                    status_code = 400
-                    payload = {'error_message':'must provide blob_type'}
-                blob_type = user_search.pop('blob_type') 
-                valid_fields = f.validate_search_fields(blob_type,user_search)
-                if not valid_fields:
-                    status_code = 400
-                    payload = {'error_message':'Invalid blob_type or search field'}
-                # no search args beyond blob_type - return all entries for blob_type
-                elif not user_search:
+                    api_resp = Response(status_code=400,error_message='Must provide blob_type',body=None)
```

Comment:
you should make `body` a default of None, so that you don't need to specify it here.

Reply:
ok, I also have error_message default to None and status_code to 200